designer/server/src/main/scala/pl/touk/nussknacker/ui/validation/ProcessValidation.scala

```diff
@@ -164,18 +167,31 @@ class ProcessValidation(
     ValidationResult.warnings(disabledNodesWarnings)
   }
 
-  private def validateIds(displayable: DisplayableProcess): ValidationResult = {
-    val invalidCharsRegexp = "[\"'\\.]".r
+  private def validateScenarioId(displayable: DisplayableProcess): ValidationResult = {
+    val fakeCanonicalProcess =
```

Comment:
Why do you need to create a fake scenario here? Why don't you pass an id instead?

Reply:
Done, I've changed it to pass id and isFragment.